File: nomor3.c
#include <stdio.h>

int main()
{
    int numbers[] = {3,4,2,5,8,1};
    int n = sizeof(numbers) / sizeof(numbers[0]);
    int i, j, temp;

    for (i = 0; i < n-1; i++)
    {
        for(j = 0; j < n- i -1; j++){
            if (numbers[j] < numbers[j+1])
            {
                temp = numbers[j];
                numbers[j] = numbers[j+1];
                numbers[j+1] = temp;
            }
            
        }
    }
    
    printf("Urutan angka dimulai dari genap ke ganjil adalah sebagai berikut: ");
    for(i = 0; i< n ; i++){
      if( numbers[i] % 2 == 0)
      {
        printf("%d ", numbers[i]);
      } 
    }
    for(i = 0; i< n ; i++){
      if( numbers[i] % 2 != 0)
      {
        printf("%d ", numbers[i]);
      } 
    }
    return 0;
}

File: nomor3.js
function SortingEventoOdd(numbers){
    var numbers = [1,8,6,7,10] // set max dengan array pertama
    let i = 0;
     for (i = 0; i < numbers.length; i++) {
        for (let j = 0; j < numbers.length; j++){
            if (numbers[j] < numbers[j + 1]){ // pastikan jika max kurang dari array yg diurutkan
                let temp = numbers[j]
                numbers[j] = numbers[j+1]
                numbers[j+1] = temp
            }
    }
    }
    for (i = 0; i < numbers.length; i++){
        if(numbers[i] % 2 == 0){
            console.log(numbers[i]) // tampilkan value max tersebut
        }
    }
    for (i = 0; i < numbers.length; i++){
        if(numbers[i] % 2 != 0){
            console.log(numbers[i]) // tampilkan value max tersebut
        }
    }
  
  }
  
 SortingEventoOdd() // panggil fungsi dengan parameter adalah array